executor/show.go

```diff
@@ -1900,18 +1903,21 @@ func (e *ShowExec) fetchShowPumpOrDrainerStatus(kind string) error {
 	return nil
 }
 
-// createRegistry returns an ectd registry
-func createRegistry(urls string) (*node.EtcdRegistry, error) {
+// createRegistry returns an ectd registry, need to close, and error
+func createRegistry(urls string) (*node.EtcdRegistry, bool, error) {
+	if pumpClient := binloginfo.GetPumpsClient(); pumpClient != nil && pumpClient.EtcdRegistry != nil {
```

Comment:
Do we need tolerate `pumpClient` is nil?

Reply:
`createRegistry` is used in `excutor/change.go` too. Should it use the `PumpsClient`?